index.d.ts

```diff
@@ -90,13 +90,14 @@ camelcaseKeys(argv);
 //=> {_: [], fooBar: true}
 ```
 */
-declare function camelcaseKeys(
-	input: ReadonlyArray<{[key: string]: any}>,
-	options?: camelcaseKeys.Options
-): Array<{[key: string]: unknown}>;
-declare function camelcaseKeys(
-	input: {[key: string]: any},
-	options?: camelcaseKeys.Options
-): {[key: string]: unknown};
+declare function camelcaseKeys<T extends ReadonlyArray<{[key: string]: any}>>(
```

Comment:
Could we use `unknown` instead of `any` here now?

Reply:
I can't think of any reason why `unknown` wouldn't work, since the return type will still take on what's passed in. Pushing the change now.

Reply:
@sindresorhus Additional tests pushed